File: python_master_reference/17_break_and_continue.py
"""
17 - Break and Continue
break stops a loop.
continue skips the current round.
"""

for number in range(10):
    if number == 5:
        break
    print(number) #prints 0 to 4

print("Second loop")

for number in range(5):
    if number == 2:
        continue
    print(number) #prints 0, 1, 3, 4 (skips 2)

File: typescript_master_reference/17_break_and_continue.ts
/*
17 - Break and Continue
break stops a loop.
continue skips the current round.
*/
for (let number = 0; number < 10; number++) {
  if (number === 5) {
    break;
  }
  console.log(number);
}
console.log("Second loop");
for (let number = 0; number < 5; number++) {
  if (number === 2) {
    continue;
  }
  console.log(number);
}
